src/main.rs

```diff
@@ -68,7 +68,6 @@ impl<P: AsRef<Path> + Send + Sync> log::Log for Logger<P> {
                 Box::new(
                     std::fs::OpenOptions::new()
                         .create(true)
-                        .write(true)
```

Comment:
Why has this line been removed?

Reply:
A clippy warning was preventing this from being merged, see message below:

warning: src\main.rs:71: unnecessary use of `.write(true)` because there is `.append(true)`
help: src\main.rs:71: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#ineffective_open_options
note: src\main.rs:71: `#[warn(clippy::ineffective_open_options)]` on by default
help: src\main.rs:71: remove `.write(true)`
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 2.95s

Reply:
Thanks, I should have seen that in CI ... appreciate it!